libs/designsystem/form-field/src/input/input.component.spec.ts

```diff
@@ -157,3 +157,28 @@ describe('when configured with size medium', () => {
     });
   });
 });
+
+describe('ngOnInit', () => {
+  const createHost = createHostFactory({
+    component: InputComponent,
+  });
+
+  it('should emit "kirbyChange" if the native input has a value', fakeAsync(() => {
+    const spectator = createHost(`<input kirby-input size="md" value="test value"/>`);
+    const onChangeSpy = spyOn(spectator.component.kirbyChange, 'emit');
+
+    tick();
```

Comment:
Is there a better way to simulate `setTimeout` than `tick`?

Reply:
![image](https://github.com/kirbydesign/designsystem/assets/9556693/daf841bf-7d79-435b-921d-edfbaae8c8af)